src/sentry/static/sentry/app/components/alertLink.tsx

```diff
@@ -50,16 +47,16 @@ export default class AlertLink extends React.Component<Props> {
       onClick,
       withoutMarginBottom,
       openInNewTab,
+      to,
     } = this.props;
     return (
       <StyledLink
-        to={(this.props as PropsWithTo).to}
-        href={(this.props as PropsWithHref).href}
```

Comment:
As mentioned in https://github.com/getsentry/sentry/pull/17346#discussion_r395202569 - I think these need to remain.

I also think that it was intentional here to open an external link but still have it open in the same tab since it was linking to our docs.

ExternalLink might need a prop to disable opening in a new tab (default should open in a new tab).